themes/bootstrap3/js/searchbox_controls.js

```diff
@@ -144,6 +144,10 @@ VuFind.register('searchbox_controls', function SearchboxControls() {
     _textInput = document.getElementById('searchForm_lookfor');
     _resetButton = document.getElementById('searchForm-reset');
 
+    if (!_textInput || !_resetButton) {
```

Comment:
I don't think a missing reset button should cause the whole initialization to fail. Maybe we should only return if _textInput is unset, and wrap the _resetButton.addEventListener call in an `if (_resetButton) {` check. What do you think?

Reply:
@demiankatz I think you are right. Currently the _resetButton should always exist if _textInput exists so _testInput alone should be enough. But if that changes in the future it might be a good idea to check for _resetButton as well.

Besides of adding an event listener in the init method _resetButton is also used in _handleInputChange. So the check should be also made there.

Reply:
I think it could be better. I'll adjust